litellm/__init__.py

```diff
@@ -852,7 +855,28 @@ def add_known_models():
     "petals-team/StableBeluga2",
 ]
 
-ollama_models = ["llama2"]
+# Since Ollama models are local, it is inexpensive to refresh the list at startup.
```

Comment:
please don't define code in __init__

Reply:
I was leery to do this too.

Where is the best place to set a module-level global that is not constant?

Reply:
I'll push up a patch that moves it to initialize().  I could also do load_config(), which is called by initialize().

Reply:
Pushed.